playground.sql

```diff
@@ -0,0 +1,8 @@
+\c nc_news_test
```

Comment:
This file should be added to your .gitignore as it's not necessary for the project